test/support/factory.ex

```diff
@@ -382,7 +382,7 @@ defmodule Trento.Factory do
 
   def tag_factory do
     %Tag{
-      value: Faker.Beer.hop(),
+      value: Faker.UUID.v4(),
```

Comment:
We should use the `sequence` option of exmachina here
https://github.com/thoughtbot/ex_machina#overview

Reply:
It wouldn't make much of a diference but... ok, it looks prettier and we get to keep the hops :smile:

Reply:
It is to keep things meaningful. This value is regular string, not a uuid. Otherwise, we could put UUID everywhere hehe